tokenserver/tests/support.py

```diff
@@ -48,3 +51,17 @@ def get_assertion(email, audience="*", issuer='browserid.org',
 
     assertion = make_assertion(email, audience, issuer=issuer, **kwargs)
     return assertion.encode('ascii')
+
+
+class CatchErrors(object):
+    def __init__(self, app):
+        self.app = app
+        if hasattr(app, 'registry'):
+            self.registry = app.registry
+
+    @wsgify
+    def __call__(self, request):
+        try:
+            return request.get_response(self.app)
+        except (exc.HTTPException, HTTPException) as e:
+            return e
```

Comment:
What happens if we don't use this wrapper? Are there many tests that rely on those exceptions?

Is it because we don't use WebTest that we have to do this?